bleachbit/Action.py

```diff
@@ -408,15 +409,13 @@ def run_process():
                     rc = 0  # unknown because we don't wait
                     from subprocess import Popen
                     Popen(self.cmd)
-            except Exception, e:
+            except Exception as e:
                 raise RuntimeError(
                     'Exception in external command\nCommand: %s\nError: %s' % (self.cmd, str(e)))
             else:
                 if not 0 == rc:
-                    logger = logging.getLogger(__name__)
```

Comment:
We don't use `logging.config`, so we're not affected by this problem. I also did a quick test with 2 files and file level logger instances and it worked as it should.